src/features/protocol/profile-protocol.ts

```diff
@@ -0,0 +1,123 @@
+export type AboutProtocolType = {
+  gender?: string;
+  age?: number;
+  bio?: string;
+  hometown?: string;
+  website?: string;
+  socials?: { appName: string; url: string }[];
+};
+
+export type NameProtocolType = {
+  first?: string;
+  last?: string;
+  middle?: string;
+  nickname?: string;
+  maiden?: string;
+};
+
+export const profileProtocol = {
+  profile: {
+    definition: {
+      protocol: "./",
```

Comment:
The 'protocols' field is required to be a URI

Reply:
Fixed